presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,309 @@
+===========================
+Hive Security Configuration
+===========================
+
+The default security configuration of the :doc:`/connector/hive` does not use
+authentication when connecting to a Hadoop cluster. All queries are executed as
+the user who runs the Presto process, regardless of the user who submits the
+query.
+
+The Hive connector provides additional security options to support Hadoop
+clusters that have been configured to use :ref:`Kerberos
+<hive-security-kerberos-support>`.
+
+When accessing :abbr:`HDFS (Hadoop Distributed File System)`, Presto can
+:ref:`impersonate<hive-security-impersonation>` the end user who is running the
+query. This can be used with HDFS permissions and :abbr:`ACLs (Access Control
+Lists)` to provide additional security for data.
+
+.. _hive-security-kerberos-support:
+
+Kerberos Support
+================
+
+In order to use the Hive connector with a Hadoop cluster that uses ``kerberos``
+authentication, you will need to configure the connector to work with two
+services on the Hadoop cluster:
+
+* The Hive metastore Thrift service
+* The Hadoop Distributed File System (HDFS)
+
+Access to these services by the Hive connector is configured in the properties
+file that contains the general Hive connector configuration.
+
+.. note::
+
+    If your ``krb5.conf`` location is different from ``/etc/krb5.conf`` you
+    must set it explicitly using the ``java.security.krb5.conf`` JVM property
+    in ``jvm.config`` file.
+
+    Example: ``-Djava.security.krb5.conf=/example/path/krb5.conf``.
+
+Hive Metastore Thrift Service Authentication
+--------------------------------------------
+
+In a Kerberized Hadoop cluster, Presto connects to the Hive metastore Thrift
+service using :abbr:`SASL (Simple Authentication and Security Layer)` and
+authenticates using Kerberos. Kerberos authentication for the metastore is
+configured in the connector's properties file using the following properties:
+
+================================================== ============================================================ ==========
```

Comment:
Sorry, missed this one earlier. I've also asked @BrendaNoonan (one of our docs folks) to look it over if she has time. I'll take a look at the other connector docs in the meantime.